packages/metrics/src/influxdb/client.ts

```diff
@@ -187,6 +187,29 @@ export const query = <T = any>(q: string): Promise<T> => {
   }
 }
 
+export function deleteMeasurements() {
+  try {
+    return Promise.all([
+      influx.dropMeasurement('birth_reg', INFLUX_DB),
+      influx.dropMeasurement('death_reg', INFLUX_DB),
+      influx.dropMeasurement('in_complete_fields', INFLUX_DB),
+      influx.dropMeasurement('declaration_time_logged', INFLUX_DB),
+      influx.dropMeasurement('declaration_event_duration', INFLUX_DB),
+      influx.dropMeasurement('certification_payment', INFLUX_DB),
+      influx.dropMeasurement('correction_payment', INFLUX_DB),
+      influx.dropMeasurement('declarations_started', INFLUX_DB),
+      influx.dropMeasurement('declarations_rejected', INFLUX_DB)
+    ]).then(() => {
+      return {
```

Comment:
I see two `return` statements here. Was this intentional?

Reply:
It was a mistake. Rmoved return before `Promise.all([`

Reply:
But I think i can use await.